### DISCUSSION TITLE
Streetmerchant not working

### DISCUSSION THREAD

**Discussion by zlWitness466** [jef/streetmerchant#1296] [Help] [answered]
I have successfully downloaded everything for the source code to work and for some reason it will not work properly. The listing will remain NVIDIA when I didn't select for NVIDIA to be selected.

**Comment by Phootonz** [comment, marked as answer]
Copy the .env-example file to a file called .env. Edit the value for the STORES config with the stores you want to search Ex: STORES="bestbuy, amazon"

**Reply by zlWitness466** [reply]
I actually did that and it still didnt work.

**Reply by Phootonz** [reply]
it defaults to NVIDIA if there is no list provided. Maybe doublecheck a syntax error.

**Reply by zlWitness466** [reply]
Within the .env?

**Reply by Phootonz** [reply]
yeah... maybe share your config

**Reply by jef** [reply]
You'll need to update the `STORES=` variable.

**Reply by SaltySeeds** [reply]
delete the clone from your computer and re-clone it.. then repeat the process over again - this worked for me! seems like there was an issue with my prior .env file

**Comment by zlWitness466** [comment]
I can do that. I actually just left my house, so I will do that later
tonight when I get back.

**Comment by zlWitness466** [comment]
Where’s the source code for that if you don’t mind me asking?

**Reply by Phootonz** [reply]
Here is where stores are selected from the env: https://github.com/jef/streetmerchant/blob/9dc04b36388907f07da5555d19fa517ed0b207ed/src/config.ts#L359
here is the line to update:
https://github.com/jef/streetmerchant/blob/9dc04b36388907f07da5555d19fa517ed0b207ed/.env-example#L82

**Reply by zlWitness466** [reply]
I tried that and I am not sure if I am typing the syntax wrong or what but it still doesn't work.

**Reply by jef** [reply]
https://jef.codes/streetmerchant/getting-started/#customization

**Reply by snadsnad** [reply]
Make sure you're editing the "dotenv" file now and not ".env".  Sometime between yesterday and now it started ignoring the .env file and when I would attempt to start after pulling it was loading a default "dotenv" file.  I copied my existing .env to dotenv and it is back to working as expected.

**Reply by jef** [reply]
Sorry, I've fixed this as well. `.env` is now working again, but it's favored to use `dotenv`.

Thank you @snadsnad!

**Comment by zlWitness466** [comment]
Awesome, I will give that a try. Apologize for not getting back with you
guys, have had exams all week.
